workshop-administration.md

```diff
@@ -125,6 +125,32 @@ For self-organized workshops:
   be set up. For this follow the instructor guide in the lesson material.
 
 
+## Workshop preparation checklist
+
+- This checklist can be set up as an issue under 
+  https://github.com/coderefinery/coderefinery.org/ or on another 
+  repository to keep track of the progress
+  ```
+  - [ ] reserve dates
+  - [ ] decide workshop organizer
+  - [ ] (online) prepare Zoom link or (in-person) book lecture room
+  - [ ] announce (twitter, notify-me, mailing lists)
+  - [ ] team of instructors complete
+  - [ ] team of helpers complete
```

Comment:
Move this to lower - now we recruit helpers as part of registration?

Reply:
thanks, but i'm not sure what you mean. Move what to lower where?

Reply:
It didn't thread properly.  I replied by email to the comment above, and it got added to the general conversation.

Reply:
ah, yeah i wasn't thinking much about the list being chronologically ordered, but it's clearer if the "helpers complete" item is lower. Now applied